pkg/featuresupport/hamasters/aws.go

```diff
@@ -0,0 +1,12 @@
+package hamasters
+
+import (
+	"github.com/giantswarm/gsctl/pkg/featuresupport"
+)
+
+var (
+	aws = featuresupport.Provider{
+		Name:            "aws",
+		RequiredVersion: "11.5.0",
```

Comment:
This information should be taken from the API, just like with node pools. The api-spec branch adds a feature named `ha_masters`. See also https://github.com/giantswarm/api/blob/hamasters/server/endpoint/v4/info/response/features.go#L7

Reply:
You killed all my hopes and dreams :(